mender-dist-packages-building/install-armv6-cross-toolchain

```diff
@@ -238,7 +238,14 @@ function install_packages_to_toolchain_root() {
         download_pkg ${cache} ${pkg}
         echo "Extracting ${pkg} to an archive (tar.(xz|gz))...";
         ar -x ${cache}/${pkg}_*_armhf.deb data.tar.xz;
+        # List the file contents
+        tar tvf data.tar.xz
+        # Since '/arm-linux-gnueabihf/' is not in the default LIBRARY_PATH for
+        # the compiler, simply remove it
         tar --extract  --directory=${toolchain_sysroot} --file=data.tar.xz --transform='s,/arm-linux-gnueabihf,,'
+        # And also keep the same files, with the 'arm-linux-gnueabihf' path to
+        # keep 'pkg-config' happy
+        tar --extract  --directory=${toolchain_sysroot} --file=data.tar.xz
```

Comment:
Is this all it takes? :tada:

Reply:
This, alongside re-installing `pkg-config` :)

Reply:
(That was my point, that this is all i takes for`pkg-config` to work properly :+1: )

Reply:
haha, yup :smiley_cat: